cpp/src/parquet/page_index.cc

```diff
@@ -533,6 +569,21 @@ class ColumnIndexBuilderImpl final : public ColumnIndexBuilder {
     /// Decide the boundary order from decoded min/max values.
     auto boundary_order = DetermineBoundaryOrder(min_values, max_values);
     column_index_.__set_boundary_order(ToThrift(boundary_order));
+
+    /// Finalize level histogram.
```

Comment:
Ideally, we're using triple-slashes (`///`) for docstrings and double-slashes (`//`) for regular comments. Some editors may highlight them differently.